nameko/cli/commands.py

```diff
@@ -72,6 +72,10 @@ def init_parser(parser):
             help='RabbitMQ broker url')
 
         parser.add_argument(
+            '--listen',
```

Comment:
`listen` sounds a little generic to me, maybe `web-server-listen` (that's a bit long, so open to other suggestions)